code/game/objects/items/storage/bags.dm

```diff
@@ -209,6 +209,30 @@
 		))
 ////////
 
+// -----------------------------
+//          Mini Plant Bag
+// -----------------------------
+
+/obj/item/storage/bag/smallplants
```

Comment:
this can be a subtype of `/obj/item/storage/bag/plants` with the atom storage modified on init

Reply:
how would i do that

Reply:
/obj/item/storage/bag/plants/small then change things as appropriate

Reply:
Gonna implement the changes yall suggest today or tomorrow when I'm not high